developer/src/kmc-convert/test/kmn-file-writer.tests.ts

```diff
@@ -22,73 +22,32 @@ describe('KmnFileWriter', function () {
     compilerTestCallbacks.clear();
   });
 
-  describe("write() ", function () {
+  describe("writeToFile()() ", function () {
```

Comment:
```suggestion
  describe("writeToFile()", function () {
```

Here and elsewhere in the tests, for writeToFile() and write(), the parentheses are duplicated